dashboard/app/models/plc/written_enrollment_task_assignment.rb

```diff
@@ -1,44 +0,0 @@
-# == Schema Information
```

Comment:
If this is going away, then the "REVIEW_REJECTED" state needs to exist in enrollment_task_assignment. Unless we are modeling this differently